ci/check-style.sh

```diff
@@ -34,7 +34,7 @@ readonly TIMEFORMAT="... %R seconds"
 problems=""
 printf "%-30s" "Running check-include-guards:"
 time {
-  if ! find google/cloud generator -name '*.h' -print0 |
+  if ! git ls-files -z | grep -zE '\.h$' |
```

Comment:
This breaks when you `git rm` a file ... `find` won't find it, but `git ls-files` will list it.

Reply:
Clarification: What I said is only the case for an __unstaged__ deletion, which makes it less problematic.